sbi/simulators/linear_gaussian.py

```diff
@@ -4,37 +4,37 @@
 import sbi.utils as utils
 
 
-def linear_gaussian(parameters: torch.Tensor, std=1.0) -> torch.Tensor:
+def linear_gaussian(theta: torch.Tensor, std=1.0) -> torch.Tensor:
```

Comment:
we should `from torch import Tensor` and then use `Tensor` for typing. just do a search for `torch.Tensor` and replace by `Tensor` for the entire file.